backends/platform/libretro/Makefile

```diff
@@ -0,0 +1,631 @@
+ROOT_PATH := $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
+
+# Output files prefix
+TARGET_NAME = scummvm
+
+HIDE := @
+SPACE :=
+SPACE := $(SPACE) $(SPACE)
+BACKSLASH :=
+BACKSLASH := \$(BACKSLASH)
+filter_out1 = $(filter-out $(firstword $1),$1)
+filter_out2 = $(call filter_out1,$(call filter_out1,$1))
+unixpath = $(subst \,/,$1)
+unixcygpath = /$(subst :,,$(call unixpath,$1))
+
+ifeq ($(shell uname -a),)
+   EXE_EXT = .exe
+endif
+
+ifeq ($(BUILD_64BIT),)
+ifeq (,$(findstring 64,$(platform)))
+   BUILD_64BIT := 0
+else
+   BUILD_64BIT := 1
+endif
+endif
+TARGET_64BIT := $(BUILD_64BIT)
+
+LD        = $(CXX)
+AR        = ar cru
+RANLIB    = ranlib
+LS        = ls
+MKDIR     = mkdir -p
+RM        = rm -f
+RM_REC    = rm -rf
+
+# Raspberry Pi 3 (64 bit)
+ifeq ($(platform), rpi3_64)
+   TARGET   = $(TARGET_NAME)_libretro.so
+   DEFINES += -fPIC -D_ARM_ASSEM_ -DUSE_CXX11 -DARM
+   LDFLAGS += -shared -Wl,--version-script=$(ROOT_PATH)/link.T -fPIC
+   CFLAGS  += -fPIC -mcpu=cortex-a53 -mtune=cortex-a53 -fomit-frame-pointer -ffast-math
+   CXXFLAGS = $(CFLAGS) -frtti -std=c++11
+
+# Raspberry Pi 4 (64 bit)
+else ifeq ($(platform), rpi4_64)
+   TARGET = $(TARGET_NAME)_libretro.so
+   DEFINES += -fPIC -D_ARM_ASSEM_ -DUSE_CXX11 -DARM
+   LDFLAGS += -shared -Wl,--version-script=$(ROOT_PATH)/link.T -fPIC
+   CFLAGS += -fPIC -mcpu=cortex-a72 -mtune=cortex-a72 -fomit-frame-pointer -ffast-math
+   CXXFLAGS = $(CFLAGS) -frtti -std=c++11
+
+# iOS
+else ifneq (,$(findstring ios,$(platform)))
+   TARGET  := $(TARGET_NAME)_libretro_ios.dylib
+   DEFINES += -fPIC -DHAVE_POSIX_MEMALIGN=1 -DIOS
+   LDFLAGS += -dynamiclib -fPIC
+   MINVERSION :=
+
+ifeq ($(IOSSDK),)
+   IOSSDK := $(shell xcodebuild -version -sdk iphoneos Path)
+endif
+ifeq ($(platform),ios-arm64)
+  CC        = cc -arch arm64 -isysroot $(IOSSDK)
```

Comment:
Something is odd with indentation lines 64-65 vs 67-68

Reply:
fixed